google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -169,6 +169,12 @@ void pollForChanges(@NotNull final CloudDebugProcessState state) {
     }
     catch(SocketTimeoutException sto) {
       //noop, this is expected behavior.
+    } catch (GoogleJsonResponseException e) {
+      // HTTP 409 is expected when backend responds to "hanging get" either for timeout or because
+      // a result is available (which will be retrieved via the subsequent query)
+      if (e.getStatusCode() != 409) {
+        handleBreakpointQueryError(state, e);
```

Comment:
Should we log something on a debug/info level just to inform a 409 was received?